frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -23,7 +23,21 @@ type MetabaseComponentThemeKey = FlattenObjectKeys<MetabaseComponentTheme>;
 export function getMetabaseCssVariables(theme: MantineTheme) {
   return css`
     :root {
-      ${getDesignSystemCssVariables(theme)}
+      --mb-default-font-family: "${theme.fontFamily}";
```

Comment:
make it obvious which function is used for which app.

- `getMetabaseCssVariables` for the main app and
- `getMetabaseSdkCssVariables` for the SDK
So there would be no confusion what CSS custom properties is for which app.

the main app, for example, has no ability to override these colors:

https://github.com/metabase/metabase/blob/35d1f6e9b48f24c2af057d09dd52e59d5e86e9ed/enterprise/frontend/src/embedding-sdk/lib/theme/embedding-color-palette.ts#L37-L44

so, it's unnecessary to set them as CSS custom properties.